README.md

```diff
@@ -165,61 +161,96 @@ the dependencies.
 Once the build and installation steps are complete, you can start using TensorFlow 
 with nGraph backends. 
 
-Please add the following line to enable nGraph: `import ngraph`
+To enable nGraph, in your python scripts
+
+        import ngraph
 
 Note: The actual filename for the pip package may be different as it's version 
 dependent. Please check the `build/python/dist` directory for the actual pip wheel.
 
-## Using the stable upstreamed version
+You can run tests following the instructions [here](#running-tests).
 
-nGraph is being added to the TensorFlow source tree the using pull requests from 
-time to time. 
+### Option 3: Build TensorFlow Source with nGraph
 
-In order to build that version of nGraph, download the source tree as mentioned
-above and select `Y` when prompted to build with nGraph.   
+nGraph is being added to the TensorFlow source tree. When built with this option, there is **no need to separately build `ngraph-tf` or use `pip` to install the ngraph module**. With this configuration, your TensorFlow model scripts will work with nGraph without any changes. 
 
-For this final option, there is **no need to separately build `ngraph-tf` or to 
-use `pip` to install the ngraph module**. With this configuration, your TensorFlow 
-model scripts will work without any changes. 
+#### Installation
 
-Note: The version that is available in the upstreamed version of TensorFlow usually
-lags the features and bug fixes available in the `master` branch of this repository.
+1. Install [bazel](#install-bazel-for-building-tensorflow-source) and other TensorFlow dependencies. Now, clone the source of the [tensorflow] repo to your machine. 
 
-### Running tests
+     :warning: You need the following version of TensorFlow: `v1.11.0-rc2`
 
-To run the C++ unit tests,
+        git clone https://github.com/tensorflow/tensorflow.git
+        cd tensorflow
+        git checkout v1.11.0-rc2
```

Comment:
Should we change this to v1.11 too? Similar to option 1.

Reply:
Yes - this should now be changed to r1.12 (but need to have the CI run with r1.12 first)

Reply:
Ok. this one points to v1.11.0-rc2, should we change to v1.11.0 (remove the rc tag)?

Reply:
`r1.12`